language-plutus-core/src/Language/PlutusCore/Error.hs

```diff
@@ -20,7 +20,7 @@ import qualified Data.Text                     as T
 
 data NormalizationError tyname name a = BadType a (Type tyname a) T.Text
                                       | BadTerm a (Term tyname name a) T.Text
-                                      deriving (Generic, NFData)
+                                      deriving (Show, Eq, Generic, NFData)
```

Comment:
Why do we derive `Show` here? Unless you need it for a test suite, we don't want to ever use this.

Reply:
Yeah, it's indirectly needed via hunit wantin to show an error.